src/website/components/ErrorPage.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactElement } from "react";
```

Comment:
This file is copied from https://github.com/allen-cell-animated/nucmorph-colorizer/blob/main/src/routes/ErrorPage.tsx.

Reply:
![image](https://github.com/allen-cell-animated/website-3d-cell-viewer/assets/30200665/c5d5235a-a7a9-44d7-9f51-420106933f35)